### DISCUSSION TITLE
240312

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#33] [팀회의]
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
모달창 작업 
## 오늘 할 일
모달창이 아직 하나밖에 나오지 않아서 어디를 어떻게 컴포넌트로 나눌지 모르겠어서 디자인 나오는거 좀 기다려야 할것 같습니다
## 오늘의 일정 (코어 타임)
지금 당장 할 일은 없습니다.. 
##
x

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
그룹필터 UI, 기본 기능(체크 토글버튼, colorchip있으면 나오고 없으면 기본 프로필 )
## 오늘 할 일
네브바 context로 관리하기 , 캘린더 ui 수정하기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항
메인페이지 주간달력, 개인, 팀페이지 월간 달력이 있는데 현재 주간달력 따로 월간 달력 따로 따로  만들었는데 월간달력을 사용해서 주간 달력을 만들어야 하는지 궁금합니다. 
오늘 날짜에만 text색상을 다르게 지정해야 하는데 해봤는데 왜 안되는지 모르겠습니다. (코드 보여드릴 예정)

**Comment by jisunipark** [comment]
## 논의사항


## 어제 한 일
- 메인 페이지 레이아웃 ui 작업

## 오늘 할 일
- 메인 페이지 레이아웃 ui 작업 마무리
- 데이터 없는 경우 ui 구현
- 사이드바 작업 (드롭다운, 말풍선(?))

## 오늘의 일정 (코어 타임)
- 1:40 병원 (1:10 - 3:00 정도 자리 비울 것 같습니다)

## 멘토님 질문 사항
- 전체 회의 때 얘기헀던 화면 사이즈 관련 내용

**Comment by caseV12** [comment]
## 논의사항
공통 컴포넌트에 대해
## 어제 한 일
- 모달 시스템 리팩터링, 주석 달기
- 로그인 페이지 작업 진행
## 오늘 할 일
- 로그인 페이지 완성
## 오늘의 일정 (코어 타임)
2시에 치과 이슈(한 시간 넘게 걸립니다)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
- 필겸님: 모달에서 사용하는 컴포넌트
- 채빈님: 버튼 컴포넌트
